tests/conftest.py

```diff
@@ -179,8 +179,8 @@ def add_mender_conf_to_image(image, d, mender_conf):
     with open(mender_conf_tmp, "w") as f:
         f.write(mender_conf)
 
-    new_image = os.path.join(d, image)
-    shutil.copy(image, new_image)
+    new_image = os.path.join(d, image[1])
```

Comment:
Hmm, is this a fix from your last work? I don't get it?

Ahh, I see. It's because you return the two-tuple.

I think I would solve this differently tbh. So that all fixtures return a path. That way we have a uniform and consistent API. Instead extract the path from the returned full path :)